build/Helix/runtests.cmd

```diff
@@ -0,0 +1,17 @@
+robocopy %HELIX_CORRELATION_PAYLOAD% . /s
+
+::This script file is what runs on the helix machine to start our tests.
+te ProjectReunionHelix.dll /unicodeOutput:false /enableWttLogging /screenCaptureOnError /reportLoadingIssue
```

Comment:
this basically runs all the tests in ProjectReunionHelix.dll?

Reply:
yes this is the cmdline that runs on the helix machine that gets queued up. The test dll and all of its dependencies gets deployed on the machine and then it runs this command.